call-of-duty4mwmetaconfig.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "ConfigFile":"main/server.cfg",
+        "AutoMap":true,
+        "ConfigType":"kvp",
+        "ConfigFormat":"set {0} \"{1}\"",
+        "ConfigFormatRegex":"^set (?<key>.+?) \"?(?<value>.*?)\"?$"
+    },
+    {
+        "ConfigFile":"main/server.cfg",
+        "AutoMap":true,
+        "ConfigType":"kvp",
+        "ConfigFormat":"sets {0} \"{1}\"",
+        "ConfigFormatRegex":"^sets (?<key>.+?) \"?(?<value>.*?)\"?$"
+    }
```

Comment:
Why do you have the same thing twice? Or am I just blind... which is very possible.

Reply:
You are indeed blind. One has the `set` prefix, the other `sets`.

Reply:
Lmao it still took me a couple times to see the difference. Should probably get my eyes checked again.